README.md

```diff
@@ -48,48 +48,34 @@ Update `~/Config/FileSystemProviders.config` replacing the default provider with
 ```xml
 <?xml version="1.0"?>
 <FileSystemProviders>
-  <Provider alias="media" type="Our.Umbraco.FileSystemProviders.Azure.AzureBlobFileSystem, Our.Umbraco.FileSystemProviders.Azure">
-    <Parameters>
-      <add key="containerName" value="media" />
-      <add key="rootUrl" value="http://[myAccountName].blob.core.windows.net/" />
-      <add key="connectionString" value="DefaultEndpointsProtocol=https;AccountName=[myAccountName];AccountKey=[myAccountKey]"/>
-      <!--
-        Optional configuration value determining the maximum number of days to cache items in the browser.
-        Defaults to 365 days.
-      -->
-      <add key="maxDays" value="365" />
-    </Parameters>
-  </Provider>
+  <Provider alias="media" type="Our.Umbraco.FileSystemProviders.Azure.AzureBlobFileSystem, Our.Umbraco.FileSystemProviders.Azure" />
```

Comment:
I'd like to keep the existing config guide as-is and add an additional section in the readme for configuration via app settings.